### PR TITLE
2024 demo submission updates

### PR DESCRIPTION
**Pull request by derekburgess**
I have my google form linked, but can easily switch it out to whatever we want to go live. I included more of the questions we discussed in as well as content from the demos repo that is contextual to the questions.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -267,6 +267,7 @@ GEM
       unf_ext
     unf_ext (0.0.8.2)
     unicode-display_width (1.8.0)
+    webrick (1.8.1)
```
Comment (canyon289) [inline Gemfile.lock:270]: Whats this do?

File: _pages/events.markdown [hunk 2]
```diff
@@ -8,17 +8,37 @@ toc_sticky: true
...
 * BSidesCharm April 20-21
 * BSides SF May 4-5
 * RSA Sandbox May 6-9
-* DEFCON 32 August 8-12
+* DEF CON 32 August 8-12
 
 For a discount [RSA](https://www.rsaconference.com/rsac-programs/innovation/innovation-sandbox) use the code below
 ![RSA](/assets/images/RSAC 2024_AI Village_300x250_Code2.jpg)
 
+
```
Comment (canyon289) [inline _pages/events.markdown:24]: Do you want to add a section for general volunteers?

File: _pages/events.markdown [hunk 3]
```diff
@@ -8,17 +8,37 @@ toc_sticky: true
...
+The AIVillage is seeking 16 volunteer project demos for this years DEF CON! If you're a student or in your early career, we'll pay for your flights and hotel to present this.
+
+* Submit before April 25th you could present at RSA.
+* The top 8 demos will be invited to present them at the AI day Hackers on the Hill.
+* **Submissions should be made before May 1st 2024 to be considered in time for DEF CON.** 
+
+To learn more and submit your demo: **[Complete this google form](https://forms.gle/gZMe4SDJNqGWgdik7)**
+
+If you would like to volunteer to help setup or manage events: [Submit a general volunteer application here](https://forms.gle/vCrz3zpR8xHCsTtJ8)
```
Comment (canyon289) [inline _pages/events.markdown:39]: Ah its here 
```suggestion
## General Volunteer
If you would like to volunteer to help setup or manage events: [Submit a general volunteer application here](https://forms.gle/vCrz3zpR8xHCsTtJ8)
```

File: Gemfile.lock (hunk 1 above)
Comment (derekburgess) [reply Gemfile.lock:270]: Not sure, I didnt add it? I just pulled the repo and ran the docker container.

File: _pages/events.markdown (hunk 3 above)
Comment (derekburgess) [reply _pages/events.markdown:39]: Yeah, added it below the other 2.

**Review by canyon289 — APPROVED** [review]
Minor comments. Feel feel free to merge

File: Gemfile.lock (hunk 1 above)
Comment (canyon289) [reply Gemfile.lock:270]: Cool

**Pull request merged**